src/impl/seismic/kernels/SurfIntFused.hpp

```diff
@@ -31,6 +31,12 @@
 #include "data/MmXsmmFused.hpp"
 #include "FakeMats.hpp"
 
+#ifdef PP_MMKERNEL_PERF
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+#endif 
```

Comment:
Probably another argument for a global var so that we don't have to include OMP all over the place.